e2e/executable_test.go

```diff
@@ -17,19 +17,20 @@ type scorecard struct {
 		Confidence int      `json:"Confidence"`
 		Details    []string `json:"Details"`
 	} `json:"Checks"`
+	MetaData []string `json:"MetaData"`
 }
 
 var _ = Describe("E2E TEST:executable", func() {
 	Context("E2E TEST:Validating executable test", func() {
 		It("Should return valid test results for scorecard", func() {
 
-			file, _ := ioutil.ReadFile("../bin/results.json")
+			file, err := ioutil.ReadFile("../bin/results.json")
 
-			data := scorecard{}
+			Expect(err).Should(BeNil())
 
-			err := json.Unmarshal([]byte(file), &data)
+			data := scorecard{}
 
-			Expect(err).Should(BeNil())
+			err = json.Unmarshal(file, &data)
```

Comment:
Can we validate that metadata value is openssf in test ?

Reply:
Yes, will do that. Thanks